data/data_reader.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+from compound.compound import Compound
+
+
+class DataReader:
+    train_file = './data/data_set/dorothea_train.data'
+    train_labels_file = './data/data_set/dorothea_train.labels'
+    valid_file = './data/data_set/dorothea_valid.data'
+    valid_labels_file = './data/data_set/dorothea_valid.labels'
+    test_file = './data/data_set/dorothea_test.data'
+
+    def _convert_active(self, label: int) -> int:
+        if label == 1:
+            return 1
+        elif label == -1:
+            return 0
+        else:
+            return None
+
+    def _read_labels(self, labels_file: str) -> List[int]:
+        labels = []
+
+        with open(labels_file, 'r') as reader:
+            for line in reader.readlines():
+                labels.append(int(line))
+
+        return labels
+
+    def _read_data(self, data_file: str, labels: List[int] = None) -> List[Compound]:
+        compounds = []
+
+        with open(data_file, 'r') as reader:
+            for line in enumerate(reader.readlines()):
```

Comment:
a nie dam ci jeszcze :D Co byś powiedział na `for index, line in enumerate(reader.readlines()):` i zamiast `line[1]` dawać `line` i zamiast `line[0]` dawać index?

Reply:
no dobra :D